data.py

```diff
@@ -0,0 +1,23 @@
+import mysql.connector
+#to add time 
+import datetime
+
+mydb=mysql.connector.connect(
+host="localhost",
```

Comment:
All secrets should be in environment variables

Reply:
Yeah sir .Thanks for your valuable feedback it helped me a lot i will improvise the formats next time.